splash/qtutils.py

```diff
@@ -166,7 +166,7 @@ def qt2py(obj, max_depth=100):
 
     # print(obj, obj.__class__)
 
-    if isinstance(obj, QString):
+    if isinstance(obj, (QString,QByteArray)):
         return unicode(obj)
```

Comment:
Hmm, I'm not sure returning unicode(obj) is correct for QByteArray. It will likely try to decode the data using some encoding (latin1?), and this is incorrect if the data uses different encoding. Based only on QByteArray name, I think it should be bytes(obj), but I haven't tried it.

Where are these QByteArray objects from? Could you please add a test with non-latin text and a value which ends in a QByteArray?